controller/controller.js

```diff
@@ -74,4 +75,18 @@ exports.postComment = (req, res, next)=>{
     res.status(201).send({postedComment})
     })
     .catch(next)
-}
\ No newline at end of file
+}
+exports.patchArticleVote = (req, res, next)=>{
+    const {article_id} = req.params
+    const {inc_votes} = req.body
+
+
+    updateVote(article_id, inc_votes)
+    .then((updatedArticle)=>{
+        if(!updatedArticle){
+        res.status(404).send({error: 'Article not found'})
+        }
+        res.status(200).send({ article: updatedArticle })
+        })
```

Comment:
These should be handled by your error handlers - not here in the controller. 

I dont think this actually does anything, as you've already rejected an empty array in your model. You'll maybe get an empty object, but this isnt a falsy value. 

I think your code would run the same without this if statement.m